src/plugin/plugin.ts

```diff
@@ -66,12 +84,26 @@ export const newUnimplementedDestination = (): DestinationClient => {
   };
 };
 
-export const newPlugin = (name: string, version: string, newClient: NewClientFunction): Plugin => {
+const defaultBuildTargets: BuildTarget[] = [
+  { os: 'linux', arch: 'amd64' },
```

Comment:
This matches our current supported platforms https://github.com/cloudquery/cloudquery/blob/d76f083b5027efd579d1432f59c1d8955182d0a1/.github/workflows/release_plugin.yml#L93